pkg/antctl/command_message.go

```diff
@@ -36,6 +36,13 @@ func generate(cd *commandDefinition, args map[string]string, code int) error {
 		return fmt.Errorf("NotFound: %s \"%s\" not found", cd.use, args["name"])
 	case http.StatusInternalServerError:
 		return fmt.Errorf("InternalServerError: Encoding response failed for %s", cd.use)
+	case http.StatusBadRequest:
+		{
+			if len(args["name"]) > 0 && len(args["namespace"]) == 0 {
```

Comment:
Same as previous comment, not good to have specific logic in a generic function. StatusBadRequest is a generic error, we cannot assume it's always caused by not setting namespace, for example what if a resource doesn't have namespace argument(I'm not suggesting adding more conditions to ensure it, this is not a good place to check it).

Reply:
> Same as previous comment, not good to have specific logic in a generic function. StatusBadRequest is a generic error, we cannot assume it's always caused by not setting namespace, for example what if a resource doesn't have namespace argument(I'm not suggesting adding more conditions to ensure it, this is not a good place to check it).

I agree with that. It's not good to do so, thus it took some time for me to think about how to generate a specific error message without a special judge. And finally, I committed this because there seems to be no better ways. 

I will restore the original code. This problem would be solved in the next issue.